ai2thor/controller.py

```diff
@@ -545,8 +572,8 @@ def scenes_in_build(self):
 
         return self._scenes_in_build
 
-    def reset(self, scene="FloorPlan_Train1_1", **init_params):
-        if re.match(r"^FloorPlan[0-9]+$", scene):
+    def reset(self, scene, **init_params):
```

Comment:
Seems like forcing people to specify the scene here will cause backwards compatibility issues. Can we change this to `scene: str = "default"` or something and grab the first scene in the build as discussed above?

Reply:
Similar to all other `init_params` that work by reusing the latest update if they have not been changed, I fixed this by setting `scene=None` by default, where if `scene=None`, reuse the last used scene.